Symfony/CS/Fixer.php

```diff
@@ -311,6 +312,48 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
         return $fixInfo;
     }
 
+    /**
+     * @param \SplFileInfo     $file
+     * @param Tokens           $tokens
+     * @param FixerInterface[] $fixers
+     *
+     * @return string[]
+     */
+    public function fixFileTokens(\SplFileInfo $file, Tokens $tokens, array $fixers)
```

Comment:
check, done; 7cd692ab8065798f0eaa44958d26dfd79c48fb0f
all changes to `Fixer.php` have been reverted